src/AdoNet/Orleans.Reminders.AdoNet/Orleans.Reminders.AdoNet.csproj

```diff
@@ -0,0 +1,86 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <PackageId>Microsoft.Orleans.Reminders.AdoNet</PackageId>
+    <Title>Microsoft Orleans Reminders AdoNet Providers</Title>
+    <Description>Microsoft Orleans reminders provider for AdoNet Providers</Description>
```

Comment:
As previously mentioned about "AdoNet" -> "ADO.NET", here too. I suppose in the other packages also.